### PR TITLE
FIX-BESTPRACTICES: drone-ubuntu-vm

### PR DESCRIPTION
**Pull request by akhilthomas011**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: application-workloads/drone/drone-ubuntu-vm/azuredeploy.json [hunk 1]
```diff
@@ -50,6 +50,13 @@
         "description": "Type of authentication to use on the Virtual Machine. SSH key is recommended."
       }
     },
+    "vmSize":{
+      "type": "string",
+      "defaultValue": "Standard_A3",
+      "metadata": {
+        "description": "Username assigned to new local user on VM"
```
Comment (bmoore-msft) [inline application-workloads/drone/drone-ubuntu-vm/azuredeploy.json:57]: update description

File: application-workloads/drone/drone-ubuntu-vm/azuredeploy.json [hunk 2]
```diff
@@ -50,6 +50,13 @@
         "description": "Type of authentication to use on the Virtual Machine. SSH key is recommended."
       }
     },
+    "vmSize":{
+      "type": "string",
+      "defaultValue": "Standard_A3",
```
Comment (bmoore-msft) [inline application-workloads/drone/drone-ubuntu-vm/azuredeploy.json:55]: use the v2 series for the defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/drone/drone-ubuntu-vm/azuredeploy.json (hunk 1 above)
Comment (badri-spektra) [reply application-workloads/drone/drone-ubuntu-vm/azuredeploy.json]: updated

File: application-workloads/drone/drone-ubuntu-vm/azuredeploy.json (hunk 2 above)
Comment (badri-spektra) [reply application-workloads/drone/drone-ubuntu-vm/azuredeploy.json]: changed to v2 series

**Pull request merged**